sympy/core/compatibility.py

```diff
@@ -488,6 +488,14 @@ def default_sort_key(item, order=None):
     The order of terms obtained when using these keys is the order that would
     be obtained if those terms were *factors* in a product.
 
+    It is not useful to directly sort expressions based on their complexity
```

Comment:
Yes it seems pretty better than the previous one. I have made some modifications and made a PR.